### DISCUSSION TITLE
Documentation website

### DISCUSSION THREAD

**Discussion by nhedger** [rebing/graphql-laravel#1103] [Ideas]
What your take on setting up a documentation website for the project?

Happy to help setting it up.

**Comment by mfn** [comment]
The reason I didn't seek out to change anything is basically:
- I'm really a fan of a single (even if longer) README, assuming it has proper sections
  All the information is always on a single page and "find in page" allows to quickly locate things anyway; it's less of an issue to "miss things", i.e. otherwise you would have to read through the whole multi-page thing to not miss stuff, which I don't want to myself on other project and wouldn't want others have to.
- I've no resources to do anything like that
  And given the aforementioned argument, also not much drive to do that

I think the quality of the content of the docs is more important and the majority of the feedback over the years was that the docs are outdated, not clear/good enough; but that's unrelated to having a dedicated webpage and firstly would need humans doing that.

So before investing into changing to a different system and adding complexity to the project, I'd rather have ppl contribute more to the docs itself. And then take it from there if there are more from the community wishing for a dedicate site etc.

**Reply by nhedger** [reply]
Fair enough
